lektor/admin/static/js/views/EditPage.jsx

```diff
@@ -130,6 +130,29 @@ class EditPage extends RecordEditComponent {
   }
 
   saveChanges () {
+    const path = this.getRecordPath()
+    const alt = this.getRecordAlt()
+    const newData = this.getValues()
+    utils.apiRequest('/rawrecord', { json: {
+      data: newData, path: path, alt: alt },
+    // eslint-disable-next-line indent
+      method: 'PUT' }, makeRichPromise)
+      .then((resp) => {
+        this.setState({
+          hasPendingChanges: false
+        }, () => {
+          this.transitionToAdminPage('.edit', {
+            path: this.getUrlRecordPathWithAlt(path)
+          })
+          window.location.reload()
+          if (!resp.okay) {
+            window.alert(i18n.trans('EDIT_SAVED_SUCCESFULLY'))
```

Comment:
If possible I think it's better if the user doesn't have to dismiss the dialog manually. Maybe a 3 second timeout can be used?

I think there already exist classes to support similar dialog messages in the existing code.

Reply:
I really dig that idea. I'll look into that!

Reply:
I added a successDialog component in the most recent pull request to make it similar to the other dialogs. I couldn't find any of them set to auto close after a particular timeout, so I made this one work identical to the existing ones. If it's decided that a timeout feature to close automatically is a need, I figure we can revisit it.